.gitlab-ci.yml

```
@@ -1,183 +1,39 @@
-  image: ${DOCKER_PROXY_URL}/stackstate/datadog_build_system-probe_x64:0d87e00f
-  tags:
-    - sts-k8s-xl-runner
-  script:
-    - ./prebuild-datadog-agent.sh --generate-no-docker
-  artifacts:
-    paths:
-      - prebuild_artifacts/artifacts/*
-
-test_protobuf_updated:
-  extends: [ .linux_builder, .go_process_pull_agent_cache ]
-  stage: test
-  needs: [ dependencies ]
-  script:
-    - (cd $(go list -f '{{ .Dir }}' -m github.com/gogo/protobuf); make install)
-    - rake protobuf
-    - '[[ ! `git status --porcelain` ]] || (echo "ERROR: protobuf should be regenerated and committed" && git diff && false)'
-
-build_linux:
-  extends: [ .linux_builder, .go_process_pull_agent_cache ]
-  stage: build
-  needs: [ dependencies, prebuild_agent ]
-  script:
-    - export PROCESS_AGENT_VERSION=$(eval $VERSION_LOGIC)
-    - printenv
-    - rake build
-    - ./prebuild-datadog-agent.sh --install-ebpf
-  artifacts:
-    paths:
-      - $CI_PROJECT_DIR/process-agent
-      - $CI_PROJECT_DIR/ebpf-object-files
-    expire_in: 2 week
-  tags:
-    - sts-k8s-xl-runner
-  retry:
-    max: 2
-    when:
-      - always
-
-test:
-  extends: [ .linux_builder, .go_process_pull_agent_cache ]
-  stage: test
-  needs: [ dependencies, prebuild_agent ]
-  script:
-    - printenv
-    - rake ci
-  tags:
-    - sts-k8s-xl-runner
-  retry:
-    max: 2
-    when:
-      - always
-
-publish_binaries:
-  stage: publish
-  extends: [ .linux_builder, .go_process_pull_agent_cache ]
-  needs:
-    - build_linux
-  script:
-    - export PROCESS_AGENT_VERSION=$(eval $VERSION_LOGIC)
-    - ls -la $CI_PROJECT_DIR
-    - ls -la $CI_PROJECT_DIR/packaging
-    - cd $CI_PROJECT_DIR/packaging && ./publish_staging_package.sh
-
-.docker_build:
-  image: ${DOCKER_PROXY_URL}/docker:20
-  services:
-    - name: ${DOCKER_PROXY_URL}/docker:20-dind
-      alias: docker
-  variables:
-    DOCKER_DRIVER: overlay
-    DOCKER_HOST: tcp://docker:2375
-    DOCKER_TLS_CERTDIR: ""
-  before_script:
-    - docker login -u "${docker_user}" -p "${docker_password}" "${DOCKER_REGISTRY}"
-    - docker login -u "${quay_user}" -p "${quay_password}" "${QUAY_REGISTRY}"
-
-.publish_linux_docker:
-  extends: [ .docker_build ]
-  stage: publish
-  needs:
-    - build_linux
-    - prebuild_agent
-  script:
-    - export BUILD_TAG="${REPO}:${TAG}"
-    - docker build -t "${BUILD_TAG}" .
-    - ./packaging/publish_image.sh $BUILD_TAG $REPO $TAG $EXTRA_TAG
-
-publish_linux_docker:
-  extends: [ .publish_linux_docker ]
+agent_x86_64:
+  stage: triggers
   variables:
-    REPO: $STS_DOCKER_TEST_REPO
-    TAG: "$CI_COMMIT_REF_NAME-$CI_COMMIT_SHORT_SHA"
-    EXTRA_TAG: $CI_COMMIT_REF_NAME
-
-publish_k8s_docker:
-  extends: [ .publish_linux_docker ]
-  variables:
-    REPO: $STS_DOCKER_K8S_REPO
-    TAG: "$CI_COMMIT_REF_NAME"
-    EXTRA_TAG: $CI_COMMIT_SHORT_SHA
-
-publish_linux_docker_release:
-  extends: [ .publish_linux_docker ]
+    ARCH: x86_64
+    DD_ARCH: amd64
+    DOCKER_ARCH: amd64
+    LLVM_ARCH: x86_64
```

Maybe we could use just 2 env variables. LONG and SHORT are probably not the best names... BTW I will leave the choice to you this is not strictly necessary for the merge 

```suggestion
    LONG_ARCH: x86_64
    SHORT_ARCH: amd64
```

Same for arm

```suggestion
    LONG_ARCH: aarch64
    SHORT_ARCH: arm64
```

---

@Andreagit97 that doesn't solve the discrepancy for `LLVM_ARCH`:
```
intel:
LLVM_ARCH: x86_64

arm
LLVM_ARCH: arm64
```

---

nope actually we cannot because otherwise we need if else if arm uses the SHORT version while x86 uses the LONG one

---

never mind